components/layout_2020/flow/inline.rs

```diff
@@ -77,6 +81,119 @@ struct LinesBoxes {
 }
 
 impl InlineFormattingContext {
+    // This works on an already-constructed `InlineFormattingContext`,
+    // Which would have to change if/when
+    // `BlockContainer::construct` parallelize their construction.
+    #[allow(unused)]
+    pub(super) fn intrinsic_sizes(&self, layout_context: &LayoutContext) -> IntrinsicSizes {
+        struct Computation {
+            paragraph: IntrinsicSizes,
+            current_line: IntrinsicSizes,
+            current_line_percentages: Percentage,
```

Comment:
I'm not sure why this is plural.

Reply:
Maybe it shouldn’t? It’s (potentially) the sum of percentages values from multiple properties on multiple boxes.